src/Tracker/State.php

```diff
@@ -105,7 +105,7 @@ public function get($store, $name = '') {
         }
     }
 
-    function setState($store, $name = '', $value, $store_type = '', $is_perminent = false) {
```

Comment:
The default is needed as one could set a store without a name.

Reply:
I didn't get any errors like that. I couldn't find anywhere those variables weren't set already.